### ISSUE TITLE
Integrate with Wordpress

### ISSUE THREAD

**Issue by pietrosperoni** [fairdemocracy/vilfredo-client#197]
Wordpress is the most widely used Blog system and it is used by 10% of the websites. Making a plugin for wordpress could permit to people to use ask to their readers to answer a particular question

**Comment by giovanisp** [comment]
This will very likely grow into a new fairdemocracy/vilfredo-wordpress github project in the future, that will then use api.vilfredo.org

Inviato da iPad

**Comment by pietrosperoni** [comment]
indeed. Each different client will have its own project. If it's done by us it will be under fairdemocracy/
